src/styles/components/v-tooltip.css

```diff
@@ -1,6 +1,111 @@
 
 @import '../../styles/tools/ec-z-level-all.css';
 
+.ec-tooltip {
+  @apply tw-block;
+
+  /* base z-indexing for the tooltip */
+  @mixin ec-z-level-all;
+
+  &__inner {
+    @apply tw-bg-gray-2;
+    @apply tw-text-gray-8;
+    @apply tw-py-12 tw-px-16;
+    @apply tw-rounded;
+
+    max-width: 260px;
+
+    .ec-tooltip--bg-bright & {
+      @apply tw-shadow-level-1;
+      @apply tw-bg-gray-8;
+      @apply tw-text-gray-3;
+      @apply tw-border tw-border-solid tw-border-gray-6;
+    }
+  }
+
+  &__arrow {
+    @apply tw-z-tooltip;
+    @apply tw-w-0;
+    @apply tw-h-0;
```

Comment:
@MaruaEl You could group tw-h-0 and put it in the same line with tw-w-0

Reply:
Thanks will do! v